index.html

```diff
@@ -22,11 +22,17 @@
       <img src="assets/freddie.png" id="hero-img" alt="freddie mercury with his hand up" />
       <h1 id="hero-heading">We are the Champions</h1>
     </div>
-    <div class="input-section">
-      <textarea id="input-field" name="input-field" placeholder="Write your endorsement here" minlength="2"
-        required></textarea>
-      <button id="publish-button">Publish</button>
-    </div>
+    <form>
+      <div class="input-section">
+        <textarea class="input-field" id="endorsement-input" name="input-field"
+          placeholder="Write your endorsement here" rows="7" required></textarea>
```

Comment:
remove required

Reply:
resolved! https://github.com/diankita/we-are-champions_mobile-app/pull/3/commits/a737a1a9fa00f35b6849593bd2400ecb123b0878